docs/collecting-data/collecting-from-own-applications/rust-tracker/getting-started/index.md

```diff
@@ -31,7 +31,7 @@ use snowplow_tracker::Snowplow;
 Instantiate a tracker using the `Snowplow::create_tracker` function. The function takes three required arguments: `namespace`, `app_id`, `collector_url`, and one optional argument, `subject`. Tracker `namespace` identifies the tracker instance; you may create multiple trackers with different namespaces. The `app_id` identifies your app. The `collector_url` is the URI of the Snowplow collector to send the events to. `subject` allows for an optional Subject to be attached to the tracker, which will be sent with all events.
 
 ```rust
-let tracker = Snowplow::create_tracker("ns", "app_id", "https://...", None);
+let mut tracker = Snowplow::create_tracker("ns", "app_id", "https://...", None);
 ```
 
 There are additional optional arguments to configure the tracker. To learn more about configuring how events are sent, check out [this page](/docs/collecting-data/collecting-from-own-applications/rust-tracker/initialization-and-configuration/index.md).
```

Comment:
Nit: would it make sense to also give a snippet how to deinitialize the tracker using the `close_emitter()` call here? I know it's mentioned on the following page but it's kind of buried and might be missed by some users.

Reply:
Yeah, this seems like a good idea.